4. User Guides/Editing Messages/README.md

```diff
@@ -53,3 +53,7 @@ Various emoji shortcuts exists to display popular text representations as graphi
 Graphical emojis can be enabled/disabled in [Your Account](../managing-your-account/) settings under the **Messages** section.
 
 When graphical emojis are disabled, emoji keywords and shortcuts will be displayed as is  e.g. `:smile:` or `:)`.
+
+## Sending audio recordings
+
+You can send audio recordings by clicking on the microphone icon on the right of the compose message box, which will start recording audio and replace the microphone icon with a square "stop" icon. To end the audio recording, click the "stop" icon, which will display a window where you can listen to and download the recording, name it and give it an optional description, before choosing to send or cancel and discard the audio recording.
```

Comment:
Would you like a note like: "uploading on mobile can take a while depending on your mobile network speed, the audio recording won't appear in the message thread until the upload is complete"?